service/controllers/messages.go

```diff
@@ -95,8 +96,7 @@ func routes(routeObject response, w http.ResponseWriter) {
 	message = strings.ToLower(message)
 	messageArr := strings.Fields(message)
 	var a []string
-	priority := []string{"images", "image", "video", "videos", "watch", "youtube", "symptoms", "medicine", "weather", 
-						 "meaning", "google", "yahoo", "bing", "search"}
+	priority := config.Get().KeywordPriority
```

Comment:
Can we have `GetPriorityKeywords()` that returns this?  Similarly, `GetUserAgents()`.